hw1/02-palindrome.html

```diff
@@ -6,21 +6,25 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
-    <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
+  <body class="bg-primary">
+    <main class="bg-light w-25 mx-auto">
```

Comment:
You could add d-flex to the classes in main to allow the container to be flexible.  When I tested the code, I found that shrinking the size of the screen had side effects. The input box was not responsive and did not shrink. Everything else looked great.